dotCMS/src/main/webapp/html/portlet/ext/browser/view_browser.jsp

```diff
@@ -188,7 +187,7 @@ List<Language> languages = (List<Language>)request.getAttribute (com.dotmarketin
 
 <div id="statusDiv" align="left" style="width: 100%; display: none; height: 100px; overflow: auto;"></div>
 
-<div id="popups"></div>
+<div id="popups" class="site-browser__context-menus"></div>

```

Comment:
We talked about this class name, we agreed to change it to ```content-menu``` because we can't leave "attached" to site browser.